Source/WebCore/rendering/ios/RenderThemeIOS.mm

```diff
@@ -1312,7 +1313,7 @@ static void paintAttachmentProgress(GraphicsContext& context, AttachmentLayout&
     Path progressPath;
     progressPath.moveTo(center);
     progressPath.addLineTo(FloatPoint(center.x(), info.progressRect.y()));
-    progressPath.addArc(center, info.progressRect.width() / 2, -M_PI_2, info.progress * 2 * M_PI - M_PI_2, RotationDirection::Counterclockwise);
+    progressPath.addArc(center, info.progressRect.width() / 2, -M_PI_2, info.progress * 2 * std::numbers::pi - M_PI_2, RotationDirection::Counterclockwise);
```

Comment:
Can we get rid of the M_PI_2 here as well?

Reply:
I can use `piOverTwoDouble` instead, there is no corresponding constant in std::numbers. Is this what you had in mind?

Reply:
Yeah. Just seemed weird to mix.

Reply:
Ok. The last iteration took care of this. Thanks.